examples/example_plugin/example_plugin/static/example_plugin/docs/404.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
```

Comment:
Would it be feasible to `.gitignore` `examples/example_plugin/example_plugin/static/example_plugin/docs/` and make it so that we just make sure to render them before packaging and installing the example-plugin package? Basically make it work more similarly to nautobot itself?

Reply:
We could, but we don't currently package example_plugin.

Reply:
Also, it would be nice to have this here and then point to it in the docs as an example.

Reply:
I just don't like committing a bunch of generated files to git. If fixing that means we need to rework how we install the example-plugin then so be it.

Reply:
Fixed.